src/api/v1/Auth/controller.js

```diff
@@ -1,22 +1,137 @@
 const {
     sendSuccessResp,
     sendFailureResp,
-} = require('../../../utils/response.js');
+} = require("../../../utils/response.js");
+
+const { User, Admin } = require("../../../database/index.js");
+
+const { createToken } = require("../../../services/jwt.js");
+
+const { hash, compareHash } = require("../../../services/encryption.js");
+
+const roleArray = [User, Admin];
```

Comment:
this should be an roleMap instead of array , that too in full context or scope i.e. Config file

Reply:
conflicting change